### DISCUSSION TITLE
How do you use Nginx with GRPC WEB

### DISCUSSION THREAD

**Discussion by Nedimko123** [grpc/grpc-web#1373] [Q&A - General]
After some configuration I get 
`Paused on exception
RpcError: missing trailers`

But I do get the data back, though...

**Comment by sampajano** [comment]
@Nedimko123 Thanks for the feedback. :)

Nginx support is not part of grpc-web's core feature, and would be better supported by the larger community.

I'll convert this into a discussion in case anyone could help with it.

Thanks! :)

**Comment by KianAnbarestani** [comment]
This error means the underlying RPC channel isn’t getting the expected trailing metadata. Often, it's caused by a server or library quirk (e.g. in gRPC) and can be benign if the data is correctly returned. If everything works as expected, you might safely ignore it, but it's wise to check for updates or configuration tweaks in your RPC library to ensure full compliance with trailer expectations.
